client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import {
+	camelCase,
+	difference,
+	isEmpty,
+	keys,
+	map,
+	pick,
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getContactDetailsExtraCache } from 'state/selectors';
+import { updateContactDetailsCache } from 'state/domains/management/actions';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormCheckbox from 'components/forms/form-checkbox';
+
+const ciraAgreementUrl = 'https://services.cira.ca/agree/agreement/agreementVersion2.0.jsp';
+const defaultValues = {
+	legalType: 'CCT',
+	ciraAgreementAccepted: false,
+};
+
+class RegistrantExtraInfoCaForm extends React.PureComponent {
+	static propTypes = {
+		ciraAgreementAccepted: PropTypes.bool,
+		contactDetails: PropTypes.object,
+		legalType: PropTypes.string,
+		translate: PropTypes.function,
+	}
+
+	constructor( props ) {
+		super( props );
+		const { translate } = props;
+		const legalTypes = {
+			ABO: translate( 'Aboriginal' ),
+			ASS: translate( 'Association (Unincorporated)' ),
+			CCO: translate( 'Canadian Corporation' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'A Corporation' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/domain-ca-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).